misc/config_tools/library/board_cfg_lib.py

```diff
@@ -573,7 +573,7 @@ def get_common_clos_max():
             if res == 'MBA':
                 tmp_clos_max_list.append(clos_max)
             else:
-                tmp_clos_max_list.append(clos_max//2)
+                tmp_clos_max_list.append((clos_max + 1)//2)
```

Comment:
Is this list meant to hold the max CLOS, not the number of CLOS? If so, you need `(clos_max + 1) // 2 - 1` instead.

Reply:
Not sure if we will get new board xmls for ehl, tgl...etc. Seems they are either manual edited or really get even numbers in <CLOS_INFO>. I leave "floor" here for back compatibility.

Reply:
This is the number of count. It's used for checking if the <MBA_DELAY> and/or <CLOS_MASK> counts exceed the count maximum.